modules/samtools/samtools.nf

```diff
@@ -1,40 +0,0 @@
-// TODO
-
-nextflow.enable.dsl = 2
```

Comment:
Please don't delete this module either. Replacing with a simple `TODO` would be enough.

Reply:
ok! I'll restore it too